packages/@coorpacademy-components/src/atom/button-link-icon/test/fixtures/button-icon-bullet.js

```diff
@@ -4,6 +4,7 @@ export default {
     'aria-label': 'aria button',
```

Comment:
I'd update the aria-label props for osmething meaningfull
```suggestion
    'aria-label': 'See more',
```